bin/webserver/Dockerfile

```diff
@@ -50,7 +51,9 @@ RUN apt-get -y update && \
     docker-php-ext-install -j$(nproc) gd
 
 # Enable apache modules
-RUN a2enmod rewrite headers
+RUN a2enmod rewrite headers ssl deflate filter expires
```

Comment:
We need to mention this modules also inside the Readme.md under enabled modules.

Reply:
Done.